components/centraldashboard/public/components/landing-page.js

```diff
@@ -106,7 +108,7 @@ export class LandingPage extends mixinBehaviors([AppLocalizeBehavior], utilities
     }
 
     async logout() {
-        return this.buildHref(`/logout`);
+        location.href = `/logout`;
```

Comment:
What happened to the buildHref function? Did it get deleted by mistake? This is the same cause of why the buttons in the default notebook card aren't working. Might be a better idea to just add that back if it's causing issues in multiple places.
Although, now that I think about about it, that may have been a function in main-page.js, so I don't think calling it with "this" would work. So did this logout function in the landing-page.js file ever even worked?

Reply:
No the logout did not work. I tried to use the build href but it did not behave, so I looked up stackoverflow and this was the suggested approach.

Reply:
Didnt behave because it didn't exist. But yeah location.href should work.